test/environmentSettings.ts

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Not sure if this needs to have a copyright or not.  This is just a config file, and was autogenerated from WDIO when it created the starter project.